app/helpers/application_helper.rb

```diff
@@ -517,7 +475,7 @@ def current_url_params(permitted: [])
   end
 
   def app_metadata
-    { UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url }
+    { UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url, currentUser: { id: User.current.id } }
```

Comment:
We already have https://github.com/theforeman/foreman/blob/a68ffcb24319e30c1724de40f1a445f77d86c319/app/helpers/layout_helper.rb#L55.
Would we be able to use that one?
Current user, doesn't really sound as metadata (neighter UISettings though xD)

Reply:
Yes, we can use it. I think context is a good place for it, just naming is a bit off.